python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -133,22 +147,36 @@ def runtimeEvalString(this_obj, attr_name, val):
 
 
 def raw_eval(val):
+    ## Attempts to evaluate the val object and return the object it evaluates to if it is a Python object
+    ## Makes use of basic caching as we don't expect that things at this level should change.
+    from copy import deepcopy
+    if val in _eval_cache:
+        return deepcopy(_eval_cache[val])
+
     try:
+        ## Catch common strings which shouldn't be evaluated anyway and return them asif an exception occured
+        if val in ['', '.', 'new']:
+            _eval_cache[val] = val
+            return deepcopy(val)
```

Comment:
I think I agree. I'm not a fan of the typelist inference as it causes problems like this and I'm looking forward to a rethink in some of the Schema as this would be very nice to see.

I'd like to consider dropping support for direct string translating when we move to Ganga 6.2. It's not used in too many places anymore (thankfully) and those can be made into special cases. i.e. the datetime/ShareRef objects in the XML. I think it's time we drop support for ```j.backend = "Dirac"``` and equivalent it's too fragile to keep patching/fixing this when it breaks. (conversation around this can happen in another PR as it stands)